src/Arbel.ReSharper.ConfigureAwaitPlugin/META-INF/plugin.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<idea-plugin>
+    <depends>com.intellij.modules.rider</depends>
+    <idea-version since-build="181.4952.311" until-build="183.*"/>
```

Comment:
This should be `182.*`

Reply:
I've already uploaded the plugin. What would happen if it remained like this? Can it wait for the next version of the plugin (probably next wave of R#)?

Reply:
Nothing bad will happen, but as soon as 2018.2 is released, this plugin will be shown as compatible, although it won't be. Maybe just release asap :)